.circleci/config.base.yml

```diff
@@ -515,7 +515,7 @@ jobs:
             source .circleci/local_publish_helpers.sh
             changeNpmGlobalPath
             cd packages/amplify-migration-tests
-            retry yarn run migration_v5.2.0 --no-cache --maxWorkers=3 --forceExit $TEST_SUITE
+            retry yarn run migration_v5.2.0 --no-cache --maxWorkers=4 --forceExit $TEST_SUITE
```

Comment:
I don't think we can actually use all 4 cores since jest needs 1 cpu to manage the worker pool & OS; maybe its working now ? I noticed when I tried 4 , it just runs the 4th test at the end of the first 3

Reply:
Setting maxWorkers beyond CPU count - 1 would be harmful if our tests were CPU bound. But they mostly spawn CLI and wait for it asynchronously.